apps/web/src/components/Navigation/mobile-nav.tsx

```diff
@@ -0,0 +1,61 @@
+'use client';
```

Comment:
Couldn't we use already existing components from our package? I think shadcn even has a sheet component https://ui.shadcn.com/docs/components/sheet

Reply:
Didn't use any library because I needed the flexibility because our end goal is something like this [https://vuetube.missingprogram.me/](https://vuetube.missingprogram.me/) but kept it simple fr now.